e2e-tests/portal/cypress/e2e/journeys/maker-bond/bond-details.spec.js

```diff
@@ -46,8 +46,8 @@ context('Bond Details', () => {
 
       partials.errorSummaryLinks().should('have.length', TOTAL_REQUIRED_FORM_FIELDS);
 
-      pages.bondDetails.bondTypeInputErrorMessage().should('be.visible');
-      pages.bondDetails.facilityStageInputErrorMessage().should('be.visible');
+      cy.assertText(pages.bondDetails.bondTypeInputErrorMessage(), 'Error: Enter the Bond type');
```

Comment:
Really nice job upgrading this

Reply:
Thanks - could have done more in this file and others, but didn't want to bloat this PR even more